Lib/test/test_tools/test_md5sum.py

```diff
@@ -33,6 +34,7 @@ def test_noargs(self):
             out.startswith(b'd41d8cd98f00b204e9800998ecf8427e <stdin>'))
         self.assertFalse(err)
 
+    @unittest.skipIf(sys.platform == "win32", "test is broken on Windows")
```

Comment:
After looking at this closer, I think that it can fail also on non-Windows platforms with uncommon locale settings. The safer solution is to replace TESTFN with TESTFN_ASCII in `setUpClass()` and `tearDownClass()`.